examples/panics/panics.md

```diff
@@ -1,9 +1,15 @@
 # Panics
 
-Ballerina distinguishes normal errors from abnormal errors. Normal errors are handled by returning error values. Abnormal errors are handled using the panic statement. Abnormal errors should typically result in immediate program termination.
+Ballerina distinguishes normal errors from abnormal errors. (E.g., A programming bug or out of memory.) Normal errors are handled by returning error values. Abnormal errors are handled using the panic statement. Abnormal errors should typically result in immediate program termination. A panic has an associated error value. Since program terminates, it is not needed to include error type in function return like in `check` expression
```

Comment:
```suggestion
Ballerina distinguishes normal errors from abnormal errors (e.g., a programming bug or an out-of-memory error). Normal errors are handled by returning error values. Abnormal errors are handled using the panic statement. Abnormal errors should typically result in immediate program termination. Panic has an associated error value. Since the program terminates, it is not needed to include the error type in the function return like in the `check` expression.
```